spec/lib/workos/user_management_spec.rb

```diff
@@ -219,34 +219,6 @@
     end
   end
 
-  describe '.add_user_to_organization' do
```

Comment:
It seems that those methods are deprecated, right? Since now we have the organizations membership API.